<launch>

  <node pkg="apriltag_ros" type="apriltag_location" name="apriltag_location_node"  output="screen">
    <rosparam command="load" file="$(find apriltag_ros)/config/apriltag_location.yaml"/>
  </node>
</launch>
